addons/html_builder/static/src/website_builder/plugins/options/image_gallery_option_plugin.js

```diff
@@ -317,8 +317,21 @@ class ImageGalleryOption extends Plugin {
             carouselEl.removeEventListener("slid.bs.carousel", this.onCarouselSlid);
         }
         container.replaceChildren(slideshowEl);
+        const modifiedImage = images.find((img) =>
```

Comment:
What if I apply multiple options on multiple images and then I re-order the images, the current ```.find1``` function will only finds first image, I guess this is wrong, don't you think?

Reply:
Correct!! the current .find() function will only retrieve the first image in the images array that has the o_modified_image_to_save class. If multiple images have this class (e.g., when multiple options are applied to multiple images), the current logic will not address them all.
Thanks for the review will check it and update for that case.